config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-0-4.yaml

```diff
@@ -121,19 +121,21 @@ periodics:
       - "runner.sh"
       - "make"
       - "verify-book-links"
+      volumeMounts:
+      - name: cluster-lifecycle-github-token
       env:
```

Comment:
@asim-reza Sorry kind of mist this on the issue. Before this change the volume was not mounted correct?

As those jobs are working right now I would suggest to drop the volumes entirely instead of adding the volumeMOunt

Reply:
Yup yup that makes sense... I'll get on it... Wasn't sure if the secret consumed in the env var was applied ... So I didn't rosk the delete before

Reply:
the volume was used to create a secret and that secret was consumed from in the env

Reply:
i fear if i take out the volume the secret that is consumed will never be created

Reply:
I don't think that's how it works. Both the volume and the env var are directly referencing a secret.

But it's fine just drop the volumes, we'll merge and I verify it works afterwards or worst case do a follow-up PR.

Those jobs are super super low priority, so it's not an issue if they break.

Reply:
oh that makes much more sense... okay thats perfect then

Reply:
done